lib/src/handlers/request_handler.dart

```diff
@@ -4,18 +4,19 @@ import 'dart:io';
 import 'package:dio/dio.dart';
 import 'package:http_mock_adapter/http_mock_adapter.dart';
 import 'package:http_mock_adapter/src/exceptions.dart';
-import 'package:http_mock_adapter/src/adapter_interface.dart';
+import 'package:http_mock_adapter/src/interfaces.dart';
 import 'package:http_mock_adapter/src/interceptors/dio_interceptor.dart';
+import 'package:http_mock_adapter/src/response.dart';
 
 /// The handler of requests sent by clients.
 class RequestHandler<T> {
   /// An HTTP status code such as - `200`, `404`, `500`, etc.
   int statusCode;
 
   /// Map of <[statusCode], [ResponseBody]>.
-  final Map<int, ResponseBody> requestMap = {};
+  final Map<int, Responsable> requestMap = {};
```

Comment:
Would be ideal to update the documentation comment to include `Responsable` instead of `ResponseBody`.